docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+Learn more about the providers for [Web Monetization compatible wallets here](https://webmonetization.org/docs/resources/op-wallets/).
+
+#### Test Digital Wallet
+
+You have the option to use “play” money from the [Interledger Test Wallet application](https://rafiki.money/).  
+Here you can set up an account that is enabled for Web Monetization, and other Interledger functionality, without involving real money.
+
+You can learn more about the [Interledger Test Wallet here](https://rafiki.dev/integration/playground/testnet/).
+
+## Testing
+
+### Environment and Software
+
+The resources and tools that you will need to test Web Monetization depend on your desired role, and what you want to test.
+
+We’ve identified three roles you can play when using Web Monetization:
+
+- Website visitor: Wants to pay websites that are web monetized
+- Website owner: Wants to receive Web Monetization payments
+- Web developer: Wants to explore Web Monetization functionality.
+
+This test plan only focuses on the perspective of a website visitor.
+
+| Role              | Hardware                    | Digital wallet                                                             | Environment & software                                                                                                    |
+| :---------------- | :-------------------------- | :------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| A website visitor | Any PC with internet access | A Web Monetization enabled digital wallet from which payments will be made | <ul><li>Any operating system (OS)</li> <li>A chromium-based Web browser (e.g. Chrome, Edge, Brave) or Firefox </li> </ul> |
+
+### Reporting Issues
+
+1. Use any screen recording application to record the steps and results of your testing.
+2. Log bugs or issues, and monitor their status and resolution on GitHub [link](https://github.com/interledger/web-monetization-extension/issues).
+
+### Prerequisites
+
+Before you start testing, sign-up for a wallet:
+
+1. The sign-up process depends on your digital wallet provider. Learn more from the [Dependencies](#Dependencies) section.
+2. Your wallet provider will likely need to verify your identity (info: this is mandated by the laws of each country, or jurisdiction).
+3. The identity verification process depends on your wallet provider, and your location (info: this can be within hours or days).
+4. Once your wallet provider completes all sign-up and verification processes, you should be able to:  
+   a. Setup and manage your wallet accounts, currencies, balances etc.  
+   b. Obtain the wallet address or payment pointer for your digital wallet.
+
+### Test Data
+
+This table describes the different ways that you can set up a website that is monetized, and a website that isn’t monetized, so that you can test the Web Monetization extension.
```

Comment:
The tester isn't setting up any websites. "...different ways that you can set up a website that is monetized..." I don't know if a tester needs to know this information.